src/test/java/com/Project/accountManager/services/AccountServiceTest.java

```diff
@@ -0,0 +1,152 @@
+package com.Project.accountManager.services;
+
+import com.Project.accountManager.entities.Account;
+import com.Project.accountManager.entities.User;
+import com.Project.accountManager.repository.AccountRepository;
+import com.Project.accountManager.request.AccountCreateRequest;
+import com.Project.accountManager.request.AccountDepositRequest;
+import com.Project.accountManager.request.AccountWithdrawalRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class AccountServiceTest {
+
+    @Mock
```

Comment:
i don't see any test case in test result
\
<img width="967" alt="Screenshot 2024-06-09 at 18 56 45" src="https://github.com/serhatbsts/accountManager/assets/26492690/a2df7aeb-3182-4309-8131-a9539c4fa019">